plugins/netskope/icon_netskope/actions/replace_url_list_by_id/action.py

```diff
@@ -13,8 +13,26 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        data = {
-            Input.NAME: params.get(Input.NAME),
-            "data": {Input.URLS: params.get(Input.URLS), Input.TYPE: params.get(Input.TYPE)},
+        # START INPUT BINDING - DO NOT REMOVE - ANY INPUTS BELOW WILL UPDATE WITH YOUR PLUGIN SPEC AFTER REGENERATION
+        identifier = params.get(Input.ID)
+        name = params.get(Input.NAME)
+        type_ = params.get(Input.TYPE)
+        urls = params.get(Input.URLS)
+        # END INPUT BINDING - DO NOT REMOVE
+
+        response = self.connection.client.replace_url_list_by_id(
+            identifier,
+            {
+                "name": name,
+                "data": {"urls": urls, "type": type_},
+            },
+        )
+        return {
+            Output.ID: response.get("id"),
+            Output.NAME: response.get("name"),
+            Output.DATA: response.get("data"),
+            Output.PENDING: response.get("pending"),
+            Output.MODIFY_BY: response.get("modify_by"),
+            Output.MODIFY_TIME: response.get("modify_time"),
+            Output.MODIFY_TYPE: response.get("modify_type"),
```

Comment:
Is it worth adding default values here
```suggestion
            Output.ID: response.get("id"),
            Output.NAME: response.get("name"),
            Output.DATA: response.get("data"),
            Output.PENDING: response.get("pending"),
            Output.MODIFY_BY: response.get("modify_by"),
            Output.MODIFY_TIME: response.get("modify_time"),
            Output.MODIFY_TYPE: response.get("modify_type", "") etc etc,
```